### DISCUSSION TITLE
Ne kadar kazanacağız??

### DISCUSSION THREAD

**Discussion by Stendardo17** [IBAX-io/go-ibax#243] [Ideas]
![IMG_20220123_200653](https://user-images.githubusercontent.com/97855553/150689790-9eb88391-e4be-4002-ab8c-71bf4847d7ba.jpg)
Airdrops bize ne kadar kazandıracak???
Bunun cevabını merak mı ediyorsun???
Herkes fikrini açıkça ortaya koymalı...
Fikirlerinizi önemsiyoruz...

**Comment by Stendardo17** [comment]
Tahminim en düşük tutar 250 TL

**Reply by trkmnoguz** [reply]
allah bereket versin

**Comment by trkmnoguz** [comment]
300 liraya tamamım

**Comment by byblckd** [comment]
coinin fiyatı bile belli değilken ne kadar olacağını tahmin etmek :)
